x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/index.test.tsx

```diff
@@ -199,7 +205,7 @@ describe('AlertsCountPanel', () => {
         expect(wrapper.find('[data-test-subj="alertsCountTable"]').exists()).toEqual(true);
       });
     });
-    it('toggleStatus=false, hide', async () => {
+    it('alertsPageChartsEnabled is false and toggleStatus=false, hide', async () => {
```

Comment:
Great to see that you have tested with with both false and enabled feature flags 🏅